utils/generate-defconfig.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/sh
+# Create new defconfigs by only applying the changes from a standard defconfig
+set -e
+
+SCRIPT_DIR="$( dirname -- "$0" )"
```

Comment:
NIT:
```suggestion
SCRIPT_DIR="$(readlink -f $(dirname -- "$0"))"
```
Then you can remove the implicitly relative lookup when calling `./$MERGE_CONFIG` below